invenio_records_resources/services/uow.py

```diff
@@ -261,10 +262,32 @@ def __init__(self, celery_task, *args, **kwargs):
         self._celery_task = celery_task
         self._args = args
         self._kwargs = kwargs
+        self.celery_kwargs = {}
 
     def on_post_commit(self, uow):
         """Run the post task operation."""
-        self._celery_task.delay(*self._args, **self._kwargs)
+        self._celery_task.apply_async(
+            args=self._args, kwargs=self._kwargs, **self.celery_kwargs
+        )
+
+    @classmethod
+    def for_async_apply(cls, celery_task, args=None, kwargs=None, **celery_kwargs):
```

Comment:
Quick question: is this added just as a helper/util to set the defaults args to `tuple`?
I think we call this `def create` in another places, unless I am misunderstanding the need ☺️